### PR TITLE
Version command

### PR DESCRIPTION
**Pull request by misantron**

### CODE REVIEW

File: internal/cli/root.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var Version = "0.1.0"
```
Comment (enrichman) [inline internal/cli/root.go:11]: I would leave "valid" semver to actual tagged version. Let's keep a generic `0.0.0-dev` as default

```suggestion
var Version = "0.0.0-dev"
```

File: internal/cli/version.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Version",
+		Long:  `Display application version`,
+		Run:   runVersionCmd,
+	}
+}
+
+func runVersionCmd(_ *cobra.Command, _ []string) {
+	fmt.Printf("Version: %s", Version)
```
Comment (enrichman) [inline internal/cli/version.go:19]: I would add also the binary name, as most of the binaries:

```
-> % go version 
go version go1.18.3 linux/amd64
-> % git version
git version 2.38.0
```

```suggestion
	fmt.Printf("stego version %s", Version)
```

**Review by enrichman — CHANGES_REQUESTED** [review]

**Comment by enrichman** [comment]
Thanks for the PR. Would you mind star the repo if you liked it? :smile: :star2:

**Comment by misantron** [comment]
By the way I this the good idea to move application name to constant and use it instead of hardcoded string. What do you think?

**Comment by enrichman** [comment]
Fine for me, it could be useful somewhere else 😄

**Comment by misantron** [comment]
Done

**Review by enrichman — APPROVED** [review]

**Pull request merged**